README.md

```diff
@@ -40,6 +40,9 @@ override fun onCreate(savedInstanceState: Bundle?) {
 }
 ```
 
+Note that applications that need use a subclass of `android.app.Application`
+need to use `org.wpewebkit.WPEApplication` as their base class instead.
+
```

Comment:
I don't understand the sentence, do you mean something like:

"Note: any `Application` class MUST inherit from `org.wpewebkit.WPEApplication` instead of the classical `android.app.Application`" ?